src/main/java/sirius/tagliatelle/expression/MethodCall.java

```diff
@@ -73,9 +74,29 @@ public Object eval(LocalRenderContext ctx) {
                 return method.invoke(self);
             }
 
-            Object[] params = new Object[parameterExpressions.length];
-            for (int i = 0; i < parameterExpressions.length; i++) {
-                params[i] = parameterExpressions[i].eval(ctx);
+            // Calculates if the called method has a varargs Parameter or not
+            int varargsIndex = method.getParameterCount() - 1;
+            boolean isVarargsMethod = method.getParameterTypes()[varargsIndex].isArray();
+            Object[] params = new Object[method.getParameterCount()];
+
+            if (isVarargsMethod) {
+                // Fills all non-varargs parameters
+                for (int i = 0; i < varargsIndex; i++) {
+                    params[i] = parameterExpressions[i].eval(ctx);
+                }
+                // Instantiates the varargs array of the correct type
+                Object[] varargs =
+                        (Object[]) Array.newInstance(method.getParameterTypes()[varargsIndex].getComponentType(),
```

Comment:
vllt kannsch dir den Cast sogar sparen wenn du hier gleich `params[varargsIndex] = Array.newInstance(...` machst. und in der Schleife unten dann `params[varargsIndex][j- varargsIndex] = ... `

Reply:
geht net, 1 mal mit Profis 🙈